front/src/services/AuthService.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import queryString from 'query-string';
+
+const API_URL = process.env.REACT_APP_SERVER_URL;
+
+const AuthService = {
+  login: async (email, password) => {
+    const optionsToLogin = {
+      url: `${API_URL}/oauth/token`,
+      method: 'POST',
+      data: queryString.stringify({
+        username: email,
+        password: password,
+        client_id: process.env.REACT_APP_CLIENT_ID,
+        grant_type: 'password',
+      }),
+      header: { 'content-type': 'application/x-www-form-urlencoded' },
```

Comment:
same : by putting those string in a config file, you prevent typos where you need to reuse the same key/value header elsewhere